packages/react-components/react-datepicker/src/components/CalendarDay/CalendarDay.types.ts

```diff
@@ -25,18 +24,13 @@ export interface ICalendarDay {
 /**
  * {@docCategory Calendar}
  */
-export interface CalendarDayProps extends IBaseProps<ICalendarDay>, CalendarDayGridProps {
+export interface CalendarDayProps extends CalendarDayGridProps {
   /**
    * Optional callback to access the ICalendarDay interface. Use this instead of ref for accessing
    * the public methods and properties of the component.
    */
   componentRef?: React.RefObject<ICalendarDay>;
 
-  /**
-   * Customized styles for the calendar day component
-   */
-  // styles?: IStyleFunctionOrObject<CalendarDayStyleProps, CalendarDayStyles>;
-
```

Comment:
Currently most of the components already ignore the `styles` prop and if not, they already have it removed, so it's safe to remove it.